vajram/vajram-samples/src/test/java/com/flipkart/krystal/vajram/samples/StaticCallGraphTest.java

```diff
@@ -0,0 +1,87 @@
+package com.flipkart.krystal.vajram.samples;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.flipkart.krystal.vajramexecutor.krystex.VajramKryonGraph;
+import com.flipkart.krystal.visualization.StaticCallGraphGenerator;
+import com.flipkart.krystal.visualization.models.GraphGenerationResult;
+import org.junit.jupiter.api.Test;
+
+class StaticCallGraphTest {
+
+  @Test
+  void testStaticCallGraphContainsGivenStartVajram() throws Exception {
+    try (VajramKryonGraph vajramKryonGraph =
+        VajramKryonGraph.builder()
+            .loadFromPackage("com.flipkart.krystal.vajram.samples.calculator")
+            .build()) {
+
+      // Generate static call graph for A2MinusB2
+      GraphGenerationResult result =
+          StaticCallGraphGenerator.generateStaticCallGraphContent(vajramKryonGraph, "A2MinusB2");
+
+      String htmlContent = result.getHtml();
+
+      // Check if the HTML contains the main Vajram name
+      assertTrue(
```

Comment:
The testing style guide for repo is to use AssetJ assertions rather than junit assertions.
It is considered more readable. It will also be a good thing for you to learn.
Please look at other test cases and replace with assertj assertions